cfgov/unprocessed/css/pages/ask.less

```diff
@@ -3,9 +3,11 @@
    Ask CFPB pages at /ask-cfpb/.
    ========================================================================== */
 
+@readable-width: 600px;
```

Comment:
Yes. How did I miss that? Although, does it still make sense to use the local variable to convey purpose?